### PR TITLE
Move README.md to docs

### PR DESCRIPTION
**Pull request by supl**
This PR
- Moves README.md to the docs folder
- Revises a link in README.md to use URL instead of relevant path

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+As clientService is created successfully, users can use its functions to send the requests
+|Name|Request|
+|----|-------|
+|RegisterCertificate|Certificate registration|
+|RegisterContract|Contract registration|
+|ExecuteContract|Contract execution|
+|ValidateLedger|Ledger validation|
+
+The souce code in the [example](https://github.com/scalar-labs/scalardl-go-client-sdk/example) sub-folder demonstrate the details respectively.
```
Comment (josh-wong) [inline docs/README.md]: I get an error when I go to this link. Should the URL be `https://github.com/scalar-labs/scalardl-go-client-sdk/tree/main/example`?

**Review by josh-wong — APPROVED** [review]
I left one minor question. Other than that, LGTM!

File: docs/README.md (hunk 1 above)
Comment (supl) [reply docs/README.md]: My bad.
I fixed it in https://github.com/scalar-labs/scalardl-go-client-sdk/pull/26/commits/22dfd8f738fd12b4b7c1b820d5bebb6abb7a522b

**Review by josh-wong — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**